xmltodict.py

```diff
@@ -109,15 +109,18 @@ def startElement(self, full_name, attrs):
 
     def endElement(self, full_name):
         name = self._build_name(full_name)
+        save_callback = False
+
         if len(self.path) == self.item_depth:
             item = self.item
             if item is None:
                 item = (None if not self.data
                         else self.cdata_separator.join(self.data))
 
-            should_continue = self.item_callback(self.path, item)
-            if not should_continue:
-                raise ParsingInterrupted
+            save_callback = self.item_callback(self.path, item)
+            if not save_callback:
+                raise ParsingInterrupted()
+
```

Comment:
Please revert this variable name change (it seems unrelated to the rest of the change)